setup.cfg

```diff
@@ -63,6 +63,7 @@ install_requires =
     packaging>=20.0
     typing_extensions>=3.7.4,<4.2.0; python_version < "3.8"
     langcodes>=3.2.0,<4.0.0
+    types-setuptools>=57.0.0
```

Comment:
Types packages only go in `requirements.txt` for use with mypy, they're not required to run spacy.

```suggestion
```

Reply:
I thought so, but this [fails a check for compatibility between `setup.cfg` and `requirements.txt`](https://github.com/explosion/spaCy/runs/7541263540). Is there an alternative way around this?

Reply:
You have to add it as an exception in `spacy/tests/package/test_requirements.py`.